### DISCUSSION TITLE
Making keys in numeric keypad available for key bindings

### DISCUSSION THREAD

**Discussion by habruening** [zed-industries/zed#17526] [Feature Requests]
I would find it very nice if key bindings could be defined for the keys of the numeric keypad without affecting the same keys of the normal keyboard. Many people don't use the numeric keypad so much. For them these keys should be made available for key bindings.

**Comment by SomeoneToIgnore** [comment]
There's https://github.com/zed-industries/zed/issues/11948 to track this, so hopefully it lands some day.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
